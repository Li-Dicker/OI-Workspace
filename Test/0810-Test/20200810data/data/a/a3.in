10
9
78424 56062 75356 31826 94866 90872 33820 22060 59309 
25165 26238 36541 31335 19458 34852 24076 36228 37156 
9
3215 24630 34416 32459 31942 75925 97043 56547 33800 
27561 26547 43839 39826 22634 24206 46557 17904 14289 
9
40066 85754 53374 625 43314 40769 69745 67153 50815 
25285 20733 43911 3799 27158 24475 23521 14210 8650 
10
54579 28998 72281 92551 56716 98059 95010 60132 45991 32893 
34567 37795 35145 30540 14824 16040 25492 44780 10044 35068 
9
45972 94082 49924 66804 28890 70399 82159 12635 33653 
47012 20621 30073 30865 15090 31785 43880 45107 15155 
10
67158 20167 59418 88367 86156 31040 53843 52676 50171 29710 
28228 23173 33782 400 44556 12068 11914 9130 46425 8572 
10
49786 65554 62382 32843 18280 3628 52186 82383 17825 82049 
37849 47292 6248 23195 43242 27540 5436 22015 1872 13141 
10
83943 74435 41230 59632 86195 53097 34767 76631 72354 90570 
44116 4339 12629 41006 7014 41241 35225 15756 15813 5445 
10
31220 66601 25510 24146 31233 9986 72381 42770 55060 79107 
17569 26197 2190 14316 8626 27138 6321 44060 29093 34573 
9
19604 12590 52469 15605 5443 3691 52082 67860 52079 
43573 33541 35593 45617 9755 37991 43377 32689 44768 
