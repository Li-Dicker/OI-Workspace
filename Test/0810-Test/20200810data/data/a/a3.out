Case #1: 770781
Case #2: 621332
Case #3: 590530
Case #4: 852670
Case #5: 717029
Case #6: 647238
Case #7: 622742
Case #8: 816120
Case #9: 567255
Case #10: 529472
